src/Hypergrid.js

```diff
@@ -89,7 +91,7 @@ var Hypergrid = Base.extend('Hypergrid', {
         );
 
         // Install shared plug-ins (those with a `preinstall` method)
-        Object.getPrototypeOf(this).installPlugins(options.plugins);
+        Hypergrid.prototype.installPlugins(options.plugins);
```

Comment:
This change seems unrelated to the PR?

Reply:
that is true.... it is unimportant & trivial & need to put it somewhere

Reply:
@nmichaud aside from this are you ready to approve?